controllers/validation/validation.go

```diff
@@ -56,3 +64,15 @@ func validateUsers(mdb mdbv1.MongoDBCommunity) error {
 
 	return nil
 }
+
+// validateArbiterSpec checks if the initial Member spec is valid
```

Comment:
```suggestion
// validateArbiterSpec checks if the initial Member spec is valid.
```

Reply:
Two other nits:
- shouldn't it be `arbiters` instead of `Member`? 
-should we drop `initial`? It also checks updates